content/en/docs/primer/definitions.md

```diff
@@ -0,0 +1,208 @@
+| Unit Test                    | ✅ Yes            | JUnit tests for a Java class                        | Verifies individual components in isolation                       |
+| Integration Test             | ✅ Yes            | API tests verifying service interaction             | Checks communication and data flow between modules                |
+| Functional/Acceptance Test   | ✅ Yes            | Selenium UI tests for a web application             | Validates end-to-end user flows against requirements              |
+| Performance Test             | ✅ Yes            | JMeter load test on an API endpoint                 | Assesses system responsiveness and stability under load           |
+| Security Scan (SAST/DAST)    | ✅ Yes            | SonarQube static analysis, OWASP ZAP dynamic scan   | Identifies vulnerabilities (often automated in CI/CD)             |
+| Contract Test                | ✅ Yes            | Pact tests between consumer and provider APIs       | Ensures API compatibility between services                        |
+| Accessibility Test           | ✅ Yes            | Automated WCAG compliance checks (e.g., Axe)        | Verifies usability for people with disabilities                   |
+| Manual Exploratory Testing   | ❌ No             | Human tester exploring application functionality    | Not an automated process, thus outside CDEvents scope             |
+| Code Linting/Formatting      | ❌ No             | `prettier --check`, `gofmt`                         | Focuses on code style and consistency, not functional correctness |
+| Dependency Vulnerability Scan| ❌ No             | `npm audit`, Snyk scan                              | Scans for known vulnerabilities in dependencies, not functional test of *your* code |
+
+---
+
+### 🧰 Tool-Specific Considerations
+
+| **Tool**         | **Definition of Test**                                         |
+|------------------|----------------------------------------------------------------|
+| Jenkins          | Defined by pipeline steps that execute test commands (e.g., `mvn test`) |
+| GitHub Actions   | Typically a job or step that runs test commands or frameworks  |
+| GitLab CI/CD     | Often a dedicated job stage (e.g., `test` stage) with test execution commands |
+| Azure DevOps     | "Test Plans" or "Test Pipelines" that run automated tests and publish results |
+| SonarQube        | Primarily for static analysis and code quality, but often integrated with test results for coverage |
+| Jest/JUnit/Pytest| Frameworks for writing and executing tests within codebases    |
+| Selenium/Cypress | Frameworks for automated browser-based functional testing      |
+
+---
+
+## 💾 Store
+
+**Storing** is the automated process of persistently saving, managing, and
+making accessible any **artifact, data, or metadata** generated or consumed
+during the SDLC. It ensures that necessary components and information are
+retrievable for future use, auditing, or deployment.
```

Comment:
Alternative:
**Storing** is the automated process of persisting, versioning, and managing access to artifacts and metadata in durable, addressable repositories. This includes initial publication, cross-region replication, artifact promotion between environments, and lifecycle management, but excludes deployment or execution of stored artifacts.